MobileWallet/TariLib/Wrappers/ByteVector.swift

```diff
@@ -64,15 +70,26 @@ class ByteVector {
     }
 
     init(byteArray: [UInt8]) {
-        self.ptr = byte_vector_create(byteArray, UInt32(byteArray.count))
+        var i: Int32 = -1
+        let err = UnsafeMutablePointer<Int32>(&i)
+        self.ptr = byte_vector_create(byteArray, UInt32(byteArray.count), err)
+        if (i != 0) {
+            // problem
```

Comment:
I don't think `init` can throw exceptions, any idea of how to handle a problem here?

Reply:
According to the docs, init can throw. 

There's a number of problems, though was going to start with just a generic exception for now.